cmd/benchttp/main.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+)
+
+// errUsage reports an incorrect usage of the benchttp command.
+var errUsage = errors.New("usage")
+
+func main() {
+	if err := run(); err != nil {
+		fmt.Println(err)
+		if errors.Is(err, errUsage) {
+			flag.Usage()
+		}
+		os.Exit(1)
+	}
+}
+
+func run() error {
+	commandName, nextArgs, err := shiftArgs(os.Args[1:])
```

Comment:
Nitpick : 
```suggestion
	command, options, err := shiftArgs(os.Args[1:])
```
to mirror the ofter used syntax 

```sh
program-example [command] <options>
```

Reply:
Done, although `comm
and` conflicts with `type command interface` below